BlueDBOnDisk/src/main/java/io/bluedb/disk/file/FileManager.java

```diff
@@ -135,6 +139,41 @@ public static void moveFile(Path src, BlueWriteLock<Path> lock) throws BlueDbExc
 		}
 	}
 
+	public static void moveWithoutLock(Path src, Path dst) throws BlueDbException {
+		try {
+			dst.toFile().mkdirs();
+			Files.move(src, dst, StandardCopyOption.REPLACE_EXISTING);
+		} catch (IOException e) {
+			e.printStackTrace();
+			throw new BlueDbException("trouble moving file from "  + src.toString() + " to " + dst.toString() , e);
+		}
+	}
+
+	public static void copyFileWithoutLock(Path src, Path dst) throws BlueDbException {
+		try {
+			dst.toFile().mkdirs();
+			Files.copy(src, dst, StandardCopyOption.REPLACE_EXISTING);
+		} catch (IOException e) {
+			e.printStackTrace();
+			throw new BlueDbException("Can't copy '" + src + "' to '" + dst + "'.", e);
+		}
+	}
+
+	public static void copyDirectoryWithoutLock(Path src, Path dst) throws BlueDbException {
+		dst.toFile().mkdirs();
+		for (File file: src.toFile().listFiles()) {

```

Comment:
if src.toFile() is not a directory, line 164 will give us an NPE. I think it'd be good to check that it's a directory before we go into the loop. Safety first kids.

Reply:
done